src/chunk_dispatch_state.c

```diff
@@ -92,7 +93,18 @@ chunk_dispatch_exec(CustomScanState *node)
 		 * won't pick it up.
 		 */
 		if (cis->arbiter_indexes != NIL)
+		{
+		   /* In PG11 several fields were removed from the ModifyTableState node and
+			* ExecInsert function nodes, as they were redundant. (See:
+			* https://github.com/postgres/postgres/commit/ee0a1fc84eb29c916687dc5bd26909401d3aa8cd).
+			*/
+#if PG96 || PG10
 			state->parent->mt_arbiterindexes = cis->arbiter_indexes;
+#else
+			Assert(IsA(state->parent->ps.plan, ModifyTable));
+			((ModifyTable *) state->parent->ps.plan)->arbiterIndexes = cis->arbiter_indexes;
```

Comment:
Can we use this line in all versions?

Reply:
I kinda don't want to futz with things in older versions if it can be avoided, so will not do that for now.